dbt/models/default/schema.yml

```diff
@@ -151,6 +303,270 @@ models:
       # TODO: Validation is catching obvious outliers
       # TODO: Joins with other tables produce consistent number of rows
   - name: default.vw_pin_universe
+    columns:
+      - name: y_3435
+        description: PIN location derived from the centroid of the largest polygon associated with the PIN
+      - name: x_3435
+        description: PIN location derived from the centroid of the largest polygon associated with the PIN
+      - name: ward_num
+        description: Ward ID for municipalities with wards
+        meta:
+          notes: currently only Chicago and Evanston
+      - name: township_code
+        description: Numeric code identifying the Cook County township of a given PIN
+      - name: tieback_proration_rate
+        description: Proration rate for a given PIN
+        meta:
+          notes: Some buildings sit across multiple PINs. This number is intended to capture the split in building value
+      - name: tax_special_service_area_num
+        description: Tax ID of the special service area of a given PIN, if applicable
+        meta:
+          notes: Taken from Cook County GIS shapefiles
+      - name: tax_code
+        description: Identifier for the unique combination of taxing districts which impose a levy on any given PIN
+      - name: school_unified_district_geoid
+        description: Unified school district ID for a given PIN
+        meta:
+          notes: CPS is included as a unified district. Taken from Cook County GIS shapefiles
+      - name: school_secondary_district_geoid
+        description: Secondary school district ID for a given PIN
+        meta:
+          notes: For CPS, secondary school attendance boundaries are used. Taken from Cook County GIS shapefiles
+      - name: school_elementary_district_geoid
+        description: Elementary school district ID for a given PIN
+        meta:
+          notes: For CPS, elementary school attendance boundaries are used. Taken from Cook County GIS shapefiles
+      - name: prop_address_zipcode_2
+        description: Property street address
+        meta:
+         notes: Not the address of the taxpayer/property owner
```

Comment:
**issue (blocking):** The single-spacing of those `notes` field starting here is causing all columns below to parse incorrectly in the docs.

```suggestion
      - name: prop_address_zipcode_2
        description: Property street address
        meta:
          notes: Not the address of the taxpayer/property owner
```

Reply:
Oops, good catch! I fixed this in 4026e44ef0186a8f1e75639f108f6f95cac22edd and also did a quick grep to confirm there are no other instances of it.